missions/LCU_Missions.txt

```diff
@@ -1402,7 +1737,7 @@ lcu_slot5_branch = {
 			OR = {
 				region = moluccas_region
 				region = australia_region
-				region = indonesia_region
+				region = indo_china_region
```

Comment:
also don't remember changing this one either

Reply:
Yep, better to return this one back

Reply:
Ah, was the opposite way around... Originally missions had `indo_china_region` despite the tooltip describing otherwise. Might be git doing something weird and reversing that change locally.

Anywho, correct way to have it is with `indonesia_region`.
```suggestion
				region = indonesia_region
```